notebooks/1.0-SB-querying-vertically-partitioned-data-simplest-case.ipynb

```diff
@@ -0,0 +1,291 @@
+       "      <td>200</td>\n",
+       "      <td>http://example.org/ont/transaction-log/Transac...</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>http://example.org/data/transaction/124</th>\n",
+       "      <td>http://example.org/ont/transaction-log/Transac...</td>\n",
+       "      <td>2015-10-16T10:22:24</td>\n",
+       "      <td>http://example.org/data/server/B</td>\n",
+       "      <td>200</td>\n",
+       "      <td>http://example.org/ont/transaction-log/Transac...</td>\n",
+       "    </tr>\n",
+       "  </tbody>\n",
+       "</table>\n",
+       "</div>"
+      ],
+      "text/plain": [
+       "                                           http://www.w3.org/1999/02/22-rdf-syntax-ns#type  \\\n",
+       "http://example.org/data/transaction/123  http://example.org/ont/transaction-log/Transac...   \n",
+       "http://example.org/data/transaction/124  http://example.org/ont/transaction-log/Transac...   \n",
+       "\n",
+       "                                        http://example.org/ont/transaction-log/processedAt  \\\n",
+       "http://example.org/data/transaction/123                                2015-10-16T10:22:23   \n",
+       "http://example.org/data/transaction/124                                2015-10-16T10:22:24   \n",
+       "\n",
+       "                                        http://example.org/ont/transaction-log/processedBy  \\\n",
+       "http://example.org/data/transaction/123                   http://example.org/data/server/A   \n",
+       "http://example.org/data/transaction/124                   http://example.org/data/server/B   \n",
+       "\n",
+       "                                        http://example.org/ont/transaction-log/statusCode  \\\n",
+       "http://example.org/data/transaction/123                                               200   \n",
+       "http://example.org/data/transaction/124                                               200   \n",
+       "\n",
+       "                                           http://www.w3.org/1999/02/22-rdf-syntax-ns#type  \n",
+       "http://example.org/data/transaction/123  http://example.org/ont/transaction-log/Transac...  \n",
+       "http://example.org/data/transaction/124  http://example.org/ont/transaction-log/Transac...  "
+      ]
+     },
+     "execution_count": 11,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "client = VerticalQueryClient([graph_a, graph_b])\n",
+    "client.query()"
+   ]
+  }
+ ],
+ "metadata": {
+  "kernelspec": {
+   "display_name": "Python 3",
+   "language": "python",
+   "name": "python3"
+  },
+  "language_info": {
+   "codemirror_mode": {
+    "name": "ipython",
+    "version": 3
+   },
+   "file_extension": ".py",
+   "mimetype": "text/x-python",
+   "name": "python",
+   "nbconvert_exporter": "python",
+   "pygments_lexer": "ipython3",
+   "version": "3.6.10"
+  }
+ },
+ "nbformat": 4,
+ "nbformat_minor": 4
+}
```

Comment:
I see you're using concat. Does concat also take care of matching the ids? I can imagine that it knows how to join the records based on their index but I'm not sure.

Reply:
Yeah it uses the index. This only works because I do `df = df.set_index('subj', drop=True)` a few lines before. If this becomes real code we should make this a bit more explicit.